### ISSUE TITLE
Split platform components rule not parsing imports

### ISSUE THREAD

**Issue by gabmontes** [Intellicode/eslint-plugin-react-native#130]
The `split-platform-components` rule only parses the components on destructuring assignments taken from the `React` variable.

One common practice is to import the components directly from `react-native`:

```js
import React from 'react'
import { ActivityIndicatiorIOS } from 'react-native' // Import pattern not detected

export default function Hello() {
  return <ActivityIndicatiorIOS/>
}
```

Please consider adding that pattern to the rule.

**Comment by Intellicode** [comment]
Fixed by https://github.com/Intellicode/eslint-plugin-react-native/pull/131
